kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -178,10 +170,8 @@
       clear() {
         this.$data = this.$options.data();
       },
-      open() {
-        this.$broadcast('open');
-      },
       close() {
+        this.$emit('close');
         this.$broadcast('close');
```

Comment:
@indirectlylit Actually, it is necessary. Broadcast doesn't seem to emit it within its own scope, so it won't close from inside the modal unless emit is also present